pkg/models/routers/routers.go

```diff
@@ -376,6 +377,17 @@ func (c *routerOperator) createOrUpdateRouterWorkload(namespace string, publishS
 	return nil
 }
 
+func (c *routerOperator) appendArg(args []string, arg string) []string {
```

Comment:
This function seems like a tool function. So how about making it be a normal function instead of being part of `routerOperator`. Further more, it would be better if you could provide a unit test for this function.

Reply:
accept

Reply:
I have modified, please review it.